### ISSUE TITLE
Mark breakdowns 

### ISSUE THREAD

**Issue by reidmcy** [UC-MACSS/persp-model#329]
Until recently we have been able to see the median and mean for each assignment. I and many other people have found this useful. Could you please reenable this.

Also if possible, could we get some kind of breakdown of marks for the class as a whole, once all the grades are together.

Thank you for a great class,
Reid

**Comment by bensoltoff** [comment]
I'll check the settings in chalk. Should have more info once final grades are completed early next week

**Comment by bensoltoff** [comment]
Individual assignments should show the class averages now, it looks like for some of them the option to show the statistics wasn't checked

**Comment by reidmcy** [comment]
Thank you

**Comment by sixisxu** [comment]
Hi Dr. Soltoff,
Sorry to bother you over spring break, but I'm wondering if we could get
feedback for our grades? My expected grade for PS7 is higher than what I
got since I've been put a lot of effort into it and I got a score above
median for every other psets. I just want to make sure where I did wrong?
(Name: Ningyin Xu, Cnetid: nyxu) Thank you!
Best,
Ariel

On Mar 18, 2017 9:44 AM, "Benjamin Soltoff" <notifications@github.com>
wrote:

Individual assignments should show the class averages now, it looks like
for some of them the option to show the statistics wasn't checked

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/UC-MACSS/persp-model/issues/329#issuecomment-287550571>,
or mute the thread
